library/psa_crypto.c

```diff
@@ -103,13 +103,21 @@ typedef enum {
 } mbedtls_psa_crypto_subsystem;
 
 /* Initialization flags for global_data::initialized */
-const uint8_t PSA_CRYPTO_SUBSYSTEM_DRIVER_WRAPPERS_INITIALIZED = 0x01;
-const uint8_t PSA_CRYPTO_SUBSYSTEM_KEY_SLOTS_INITIALIZED       = 0x02;
-const uint8_t PSA_CRYPTO_SUBSYSTEM_TRANSACTION_INITIALIZED     = 0x04;
+#define PSA_CRYPTO_DEF_SUBSYSTEM_DRIVER_WRAPPERS_INITIALIZED    0x01
+#define PSA_CRYPTO_DEF_SUBSYSTEM_KEY_SLOTS_INITIALIZED          0x02
+#define PSA_CRYPTO_DEF_SUBSYSTEM_TRANSACTION_INITIALIZED        0x04
+
+const uint8_t PSA_CRYPTO_SUBSYSTEM_DRIVER_WRAPPERS_INITIALIZED =
+    PSA_CRYPTO_DEF_SUBSYSTEM_DRIVER_WRAPPERS_INITIALIZED;
```

Comment:
We've now got both `#define` and `const uint8_t`, whereas we originally just had only the `#define`s. IMO it's not worth the extra set of identifiers for the small amount of typing we get, and I'd rather we just rolled back some of `a5189c49fbad6e5700e7a0807750a50ff21898b7`

Reply:
Ok, do you mind if I force push?